### ISSUE TITLE
windows mode minimized

### ISSUE THREAD

**Issue by rsarov** [end2endzone/libLNK#12]
Thank you, it really cool.
But I did not find how to install - windows mode minimized
in struct LinkInfo not this field

Sorry found ShellLinkHeader.ShowCommand

**Comment by end2endzone** [comment]
Yes you are right. The field `ShellLinkHeader.ShowCommand` is used to set the initial 'show state' of the corresponding executable window. This is one of the `SW_*` values described in [Microsoft ShowWindow API](https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-showwindow).

Unfortunately, the library have no public functions that exposes the `ShellLinkHeader` structure. In other words, you can't specify the ShowCommand parameter when you create a new link.

I do not have time to work on the library for the moment. I will leave the issue open until I complete the functionality. If you want to provide the functionality, I am open to push request. I am thinking about adding a new field called `showCommand` in the `LinkInfo` structure. 

In the meantime, I have also create issue #13, #14 and #15.
